src/components/OEISbar.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+    <div id="search-bar">
+        <div id="search-wrapper" @click="doSearch($event)">
+            <div id="search-input">
+                <label for="oeis">
+                    <b>Search the encyclopedia</b>
+                    (<a href="https://oeis.org">OEIS.org</a>):
```

Comment:
Completely agree and would be happy for you to add the attribute so that it will open in a new tab.

Reply:
committed

Reply:
This part looks good to me and seems to work, so please feel free to resolve this conversation if you are likewise comfortable (only not resolving it myself since you're the reviewer ;-)